src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.sdk;
+
+import java.io.IOException;
+
+import org.apache.http.HttpHost;
+
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.client.RestClient;
+import org.opensearch.client.RestClientBuilder;
+import org.opensearch.client.json.jackson.JacksonJsonpMapper;
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.opensearch.client.transport.OpenSearchTransport;
+import org.opensearch.client.transport.rest_client.RestClientTransport;
+
+/**
+ * This class creates a Client for SDK to make requests to OpenSearch
+ */
+public class SDKClient {
```

Comment:
Should this be extending some other class, like `OpenSearchClient`?

Reply:
Based on the [sample](https://opensearch.org/docs/latest/clients/java-rest-high-level/#sample-code), doesn't look like we should extend it.